backend/tests/test_main.py

```
@@ -2,15 +2,10 @@
 import sqlalchemy
-try:
-    from backend import models
-    from backend.main import app
-    from backend.database import get_session
-except ModuleNotFoundError:
-    # When tests are executed from the backend directory
-    import models
-    from main import app
-    from database import get_session
+
+from backend import models
```

Ruff (0.11.9)

`backend.models` imported but unused

Remove unused import: `backend.models`

(F401)